src/testScripts/sdkV2-scripts.ts

```diff
@@ -58,14 +58,14 @@ const CURRENT_NETWORK: Network = Network.arbitrum;
 function loadSettings(network: Network) {
   const config = CONFIGS[network];
   const provider = new ethers.providers.JsonRpcProvider(config.RPC_URL);
-  // const userSigner = new ethers.Wallet(ADMIN_ACCOUNTS.userPk, provider);
+  const userSigner = new ethers.Wallet(ADMIN_ACCOUNTS.programAdminPk, provider);
```

Comment:
don't check in this

if any user runs this script by mistake, we only want to run "getter" functions, not "set" functions to avoid overwriting anything